src/main/java/com/zendesk/maxwell/producer/MaxwellKafkaProducer.java

```diff
@@ -112,9 +115,35 @@ public MaxwellKafkaProducer(MaxwellContext context, Properties kafkaProperties,
 		this.inflightMessages = new InflightMessageList();
 	}
 
+	//getTopic updates the topic based on the topic per table format (declare by the --kafka_topic_per_table flag)
+	private void getTopic(RowMap r){
```

Comment:
better to have this function return the topic as a String (rather than mutating `this.topic`).